quantized_mesh_tile/editable_terrain.py

```diff
@@ -0,0 +1,487 @@
+# -*- coding: utf-8 -*-
+import numpy
+import os
+
+import numpy as np
+from future.utils import old_div
+
+from quantized_mesh_tile import TerrainTile
+from quantized_mesh_tile.llh_ecef import LLH2ECEF
+from quantized_mesh_tile.terrain import MAX, lerp
```

Comment:
Importing MAX seems a bit weird maybe we should attach it directly to the Terrain class.

Reply:
Sure..."the fool with the tool is still the fool"...  I'm using obviously to much auto-functions from pycharm :)